packages/core/src/Toggle/Toggle.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Toggle } from '.'
+import { render, fireEvent } from '../__test__/testing-library'
+import { colors } from '../theme'
+
+describe('Toggle', () => {
+  test('renders correctly when off', () => {
+    const { getByRole } = render(<Toggle label='Total price toggle' />)
+    const container = getByRole('switch').parentNode
+    expect(container).toHaveStyleRule('background-color', colors.gray)
+  })
+
+  test('renders correctly when on', () => {
+    const { getByRole } = render(<Toggle isOn label='Total price toggle' />)
+    const container = getByRole('switch').parentNode
+    expect(container).toHaveStyleRule('background-color', colors.blue)
```

Comment:
I think these should use palette colors; the named colors are deprecated

Reply:
Updated to use palette colors 👍